### ISSUE TITLE
Play scheduled Playlists only once (do not feed AutoDJ)

### ISSUE THREAD

**Issue by andneuma** [AzuraCast/AzuraCast#3253]
**Using Docker installation method**
Yes

**AzuraCast version**
2.2

**Host Operating System**
Ubuntu 18.04

**Describe the bug**
Thanks for providing this great tool, it helped us a lot in Covid times to stay connected with our friends and fellows all over the plays despite quarantine and lockdown! We're providing a sparse program mixed with live shows and preproduced shows we want to broadcast only once (and then never again, "one shot show"). We have one playlist containing one item serving as kind of an interlude jingle in case nothing has been scheduled / broadcasted live (rotation weight is max). Unfortunately, AutoDJ seems to broadcast all scheduled content we've ever uploaded in a non-predictive pattern despite we've scheduled it non-repeatedly (hoping to become broadcasted ONLY at the given datetime). Our frontend reports, that...

> LiquidSoap is currently shuffling from 57 uploaded songs in 38 playlists.

...which it shouldn't! As a workaround we currently disable each playlist we only want to have played once after the broadcast. We did not try to set rotation weight of each "one-shot playlist" to 1, would that be a possible solution?

**To Reproduce**
1. Setup a playlist (Basic Settings: General rotation, Weight: 3, Source: song-based, sequential) 
2. Schedule it with Date and Time setup properly, Scheduled Play Days of Week NOT checked
3. Advanced: Interrupt other songs to play at scheduled time.
4. Restart the station (which should be another issue..., we didn't manage to get the playlist broadcasted without restart!)

**Expected behavior**
Playlist should only be played once at the scheduled time.
